CHANGELOG.md

```diff
@@ -4,23 +4,22 @@
 
 ### New features
 
-#### New list modifier to add extra spacing between list items
+#### Add extra spacing between list items
 
-If a list is hard to read becuse the items run across multiple lines you can now [add extra spacing using the `govuk-list--spaced` modifier class](http://design-system.service.gov.uk//styles/typography/#adding-extra-spacing-between-list-items).
+If a [list](https://design-system.service.gov.uk/styles/typography/#lists) is hard to read because the items run across multiple lines, you can now [add extra spacing between list items](http://design-system.service.gov.uk/styles/typography/#adding-extra-spacing-between-list-items) using the new `govuk-list--spaced` class.
 
 [Pull request #1775: Add list--spaced modifier](https://github.com/alphagov/govuk-frontend/pull/1775).
 
+#### Sass now compiles faster
 
-#### Improved Sass compilation times
+GOV.UK Frontend's Sass files now compile to CSS faster, because we've changed the way that dependencies work when you import them.
 
-We've changed the way that dependencies are imported within GOV.UK Frontend to improve Sass compilation times.
+You don't need to do anything if you already import `node_modules/govuk-frontend/govuk/all` in your Sass file. Sass will automatically compile faster.
 
-If you currently import `node_modules/govuk-frontend/govuk/all`, you will automatically benefit from these improvements.
+If you import specific parts of GOV.UK Frontend in your Sass file instead, you can now make Sass compile faster by importing:
 
-If you currently import a subset of GOV.UK Frontend, you can now:
-
-- import components using their `index` file, which will avoid importing the dependencies again
-- import `node-modules/govuk-frontend/govuk/base` instead of importing `node-modules/govuk-frontend/govuk/settings/all`, `node-modules/govuk-frontend/govuk/helpers/all` and `node-modules/govuk-frontend/govuk/tools/all`
+- components using their `index` file - this will avoid importing dependencies multiple times
+- `node-modules/govuk-frontend/govuk/base` instead of `node-modules/govuk-frontend/govuk/settings/all`, `node-modules/govuk-frontend/govuk/helpers/all` and `node-modules/govuk-frontend/govuk/tools/all`
```

Comment:
This doesn't help with performance – it just simplifies the imports a little bit. Maybe it makes sense as a separate new feature, although it's ~related…

Reply:
Ah gotcha - in line with the other comment below, I've added `base` as a new feature.
I've also added `index` to the end of the code lines, because it felt odd to mention `index` specifically but then not mention it in the example - hope that works?